hs_core/templates/pages/genericresource.html

```diff
@@ -2529,7 +2530,9 @@ <h3 style="color: red" id="file-type-error"></h3>
                         }
                         setPointerEvents(true);
                     },
-                    error: function (XMLHttpRequest, textStatus, errorThrown) {
+                    error: function(XMLHttpRequest, textStatus, errorThrown){
+                        $("#div-invite-people").find(".label-danger").remove(); // Remove previous alerts
```

Comment:
The remove function is defined for the value returned by find() even if nothing is found. This will never output an error, since the remove function won't either.